### DISCUSSION TITLE
Custom DxcIncludeHandler

### DISCUSSION THREAD

**Discussion by PhoneCoolie** [amerkoleci/Vortice.Windows#378] [General]
### Discussed in https://github.com/amerkoleci/Vortice.Windows/discussions/153

I'm attempting to migrate the following code from SharpDX but cant work out the equivalent in Vortice.Windows:

UntexturedVertexShaderByteCode = ShaderBytecode.Compile(UntexturedShaderBytes, "VS", "vs_4_0");
UntexturedVertexShader = new VertexShader(DeviceFactory.D3DDevice, UntexturedVertexShaderByteCode);

UntexturedInputSignature = ShaderSignature.GetInputSignature(UntexturedVertexShaderByteCode);
UntexturedInputLayout = new InputLayout(DeviceFactory.D3DDevice, UntexturedInputSignature, elements);

Could you point me in the right direction please?

Thanks
Paul

**Comment by amerkoleci** [comment]
Please use Discussions in future, anyway:

You can take look here how to compile shader: 
https://github.com/amerkoleci/Vortice.Windows.Samples/blob/main/src/Vortice.Framework/D3D11Application.cs#L369

There are lot of samples on how to create vertex/pixel/InputLayout under D3D11, like this one:

https://github.com/amerkoleci/Vortice.Windows.Samples/blob/main/src/Direct3D11/DrawCube/Program.cs#L39
